src/videotilecontroller/VideoTileController.ts

```diff
@@ -11,6 +11,8 @@ import VideoTileState from '../videotile/VideoTileState';
  * in the [[videoTileUpdated]] callbacks.
  */
 export default interface VideoTileController {
+  keepLastFrameWhenPaused?: boolean | undefined;
```

Comment:
- Recommend creating an optional getter method, so builders cannot directly modify this property.
- Add an explanation comment like other methods.